### PR TITLE
Add CompressionStream article

### PR DESCRIPTION
**Pull request by tomayac**
Fixes `N/A` (submitted content proposal, but no Issue was created.)

Changes proposed in this pull request:

- Add the article ([staging preview](https://deploy-preview-8597--web-dev-staging.netlify.app/compressionstream/)).

### CODE REVIEW

**Comment by rachelandrew** [comment]
Not sure why you didn't get an issue, however this one isn't supported in all three engines so can't be included as an interop moment. This is one we looked at in the meeting and realised there was no Firefox support. Do you know if Firefox are currently implementing?

File: src/site/content/en/blog/compressionstream/index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+title: CompressionStream is now supported in Safari and Chrome
+subhead: >
+  Now that compression streams are supported in Chrome and Safari, they're ready for prime time!
+date: 2022-08-25
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/koW7cNlWa6BUlJsQR4wI.jpg
+alt: Clamp.
+author: thomassteiner
+tags:
+  - blog
+  - newly-interoperable
```
Comment (rachelandrew) [inline src/site/content/en/blog/compressionstream/index.md:11]: This is not interoperable

**Review by rachelandrew — CHANGES_REQUESTED** [review]
To be included this needs to have landed support in all three engines. If that is coming then add a comment as to when you think Firefox is landing support and we'll hold it till then.

**Comment by rachelandrew** [comment]
and it looks as if the script failed as I have the info in the table but it didn't get as far as creating an issue. If that happens again let us know, as the reason to have a content proposal form is to discuss proposals before making a PR! At which point I'd have asked about the actual interop status of this.

File: src/site/content/en/blog/compressionstream/index.md (hunk 1 above)
Comment (tomayac) [inline src/site/content/en/blog/compressionstream/index.md:11]:
```suggestion
```

**Comment by tomayac** [comment]
Can it be moved to dcc instead? The only place we mention `CompressionStream` is https://developer.chrome.com/articles/fetch-streaming-requests/, so it might make sense to at least have a small article like this. What do you think?

**Comment by tomayac** [comment]
Closing in favor of https://github.com/GoogleChrome/developer.chrome.com/pull/3557.

**Pull request closed without merging**